load-balance/src/LoadBalanceStep.test.tsx

```diff
@@ -0,0 +1,48 @@
+import {fireEvent, render, screen} from '@testing-library/react';
+import LoadBalanceStep, {LoadBalanceStepProperties} from "./LoadBalanceStep";
+
+let notifyKaotoCount = 0;
+let step: LoadBalanceStepProperties = {
+    parameters: [
+        {value: null}
+    ],
+};
+
+test('renders and update LoadBalance Step', () => {
+
+    render(<LoadBalanceStep
+        notifyKaoto={() => notifyKaotoCount++}
+        updateStep={(p: LoadBalanceStepProperties) => step = p}
+        step={step}
+    />);
+
+    const loadBalanceSelect = screen.getByTestId('load-balance-select');
+    expect(loadBalanceSelect).toBeInTheDocument();
+    //
+    const applyBtn = screen.getByTestId('lb-apply-button');
+    expect(applyBtn).toBeInTheDocument();
+    fireEvent.click(applyBtn);
+
+    console.log(step);
```

Comment:
What are your thoughts about logging the step during the test? I fear that at some point we might add noise to the CI

```suggestion
```

Reply:
I will delete it.